utils/netutil/network_windows.go

```diff
@@ -32,16 +32,15 @@ func assignIP(iface string, subnet net.IPNet) error {
 	return errors.Wrap(err, string(out))
 }
 
-func excludeRoute(ip net.IP) error {
-	gw, err := gateway.DiscoverGateway()
-	if err != nil {
-		return err
-	}
-
+func excludeRoute(ip, gw net.IP) error {
 	out, err := exec.Command("powershell", "-Command", "route add "+ip.String()+"/32 "+gw.String()).CombinedOutput()
 	return errors.Wrap(err, string(out))
 }
 
+func deleteRoute(ip, gw string) error {
+	return cmdutil.SudoExec("route", "delete", ip+"/32", "gw")
```

Comment:
This looks not correct as gw arg is not used. What is this last param "gw" ?

Reply:
Good catch, we don't need this this `"gw"`, removed it.